CircleciScripts/create_xcframeworks.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+
+from framework_list import frameworks
+from functions import log, run_command
+
+EXCLUDE_FROM_XCFramework = [
```

Comment:
nit, upcase constants
```suggestion
EXCLUDE_FROM_XCFRAMEWORK = [
```